src/lib/Role.svelte

```diff
@@ -8,11 +8,19 @@
 
   export let address: string;
   export let roleId: string;
+  export let disabled: boolean = false;
 
   $: short = roleId.replace(/^0x(.{4}).*(.{4})$/, '0x$1...$2');
   $: decoded = roleDict[roleId] ?? short;
 </script>
 
-<a href="/{$chain}/role/{address}/{roleId}" class="p-1 -m-1 rounded bg-gray-50 hover:bg-gray-100" title={roleId}>
+<a class:disabled={disabled} href="/{$chain}/{address}/role/{roleId}" class="p-1 -m-1 rounded bg-gray-50 hover:bg-gray-100" title={roleId}>
```

Comment:
This shorter syntax is equivalent:

```suggestion
<a class:disabled href="/{$chain}/{address}/role/{roleId}" class="p-1 -m-1 rounded bg-gray-50 hover:bg-gray-100" title={roleId}>
```